pymeasure/instruments/thorlabs/thorlabspm100usb.py

```
@@ -57,74 +62,166 @@ def wavelength(self):
+
+        self._wavelength_range = None
+
+    # For Maintaner: Is it necessary to deprecate properties that weren't exposed in the public API?
+    @property
+    def flags(self):
+        """Get the sensor flags, int.
+
+        .. deprecated:: 0.16
+           Instead use `sensor_flags`
+        """
+        warn(
```

Ruff (0.15.12)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)